.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+name: Run tests
+on: push
+
+env:
+  DATABASE_URL: postgres://root:root@localhost:5432
+
+jobs:
+  runner-job:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: root
+          POSTGRES_DB: root
+          POSTGRES_USER: root
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        working-directory: ./backend
+        run: npm ci
+
+      - name: run migrate
```

Comment:
Does not alter the functionality, but could all the step names use the same format :D Now there's some step names that start lower-case and some that start with upper-case :D